### DISCUSSION TITLE
[VOTE] Release Apollo version 1.9.2

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4188] [General]
Hi All,

@apolloconfig/pmc This is a call for vote to release Apollo version 1.9.2.

Release Note:

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.2

Release Candidate:

 * https://github.com/ctripcorp/apollo/releases/download/v1.9.2/apollo-configservice-1.9.2-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.2/apollo-adminservice-1.9.2-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.2/apollo-portal-1.9.2-github.zip
 * sha1 checksums
   - 975c158c902717b285946c8daab5396df0e8f228  apollo-configservice-1.9.2-github.zip
   - eaa497e96dd15d396ebb7767a70752b13b55905c  apollo-adminservice-1.9.2-github.zip
   - ce8a8fe3aea2388124be1a5d1584e4fa6aa9df02  apollo-portal-1.9.2-github.zip

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1029/content/com/ctrip/framework/apollo/

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1029/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.2

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/e4516e38226b28d321135311f5fe23723ea63bc6

Guide to build the release from source:

 * https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_2212-%e9%80%9a%e8%bf%87%e6%ba%90%e7%a0%81%e6%9e%84%e5%bb%ba

Voting will start now (2022/01/01) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by JaredTan95** [comment]
+1 Release this package.

**Comment by zouyx** [comment]
+1 Release this package.

Checked
[√] All uts passed.
[√] CHANGE.md .
[√] Download links are fine.

BTW, Remember to release apollo client to [Maven Repository](https://mvnrepository.com/artifact/com.ctrip.framework.apollo/apollo-client)

**Comment by lepdou** [comment]
+1 Release this package.

Checked
[√] All uts passed.
[√] CHANGE.md .
[√] Download links are fine.

**Comment by Anilople** [comment]
+1 Release this package.

checklist:
* [x] All unit testes passed
* [x] CHANGE.md
* [x] Download links are fine

**Comment by nobodyiam** [comment]
The vote to release Apollo version 1.9.2 has passed with 4 +1 and zero -1.
